[BUG] test_resize_with_aggregate failure

**Describe the bug**
The following test is failing for me:

```pytb
======================================================================
FAIL: test_resize_with_aggregate (__main__.TestWhisper)
resize whisper file with aggregate
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_whisper.py", line 838, in test_resize_with_aggregate
    self.assertEqual(len(data_low[1]), 2)
AssertionError: 10 != 2
```

**To Reproduce**
Steps to reproduce the behavior:

```
tox -e py37,py38,py39,py310,py311,pypy3
```

(these are all the implementations I have installed, and its failing the same way for all of them)

**Expected behavior**
Tests passing ;-).

**Environment (please complete the following information):**
 - OS flavor: Gentoo Linux
 - Graphite version: n/a, whisper: tried 1.1.8, 1.1.10, git master
 - Setup type: tried Gentoo ebuild and tox in git checkout
 - Python version: tried 3.7.13, 3.8.13, 3.9.13, 3.10.4, 3.11.0b1, PyPy3.9 7.3.9

**Additional context**
n/a

I don't think that's possible, I don't have whisper installed prior to testing. Inside the venv:

```
$ type -P whisper-resize.py 
/tmp/whisper/.tox/py39/bin/whisper-resize.py
```

However, now I've found even bigger wtf: it fails when run via `tox` but passes if I run the same command inside tox's venv, i.e. (I've removed the `sh` calls in case they interfered):

```
$ tox -e py39
GLOB sdist-make: /tmp/whisper/setup.py
py39 inst-nodeps: /tmp/whisper/.tox/.tmp/package/1/whisper-1.2.0.zip
py39 installed: coverage==6.4,mock==4.0.3,six==1.16.0,whisper @ file:///tmp/whisper/.tox/.tmp/package/1/whisper-1.2.0.zip
py39 run-test-pre: PYTHONHASHSEED='3199325784'
py39 run-test: commands[0] | coverage run --branch '--include=whisper.py,test_whisper.py, bin/whisper-resize.py' test_whisper.py
................................F.........
======================================================================
FAIL: test_resize_with_aggregate (__main__.TestWhisper)
resize whisper file with aggregate
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/whisper/test_whisper.py", line 838, in test_resize_with_aggregate
    self.assertEqual(len(data_low[1]), 2)
AssertionError: 10 != 2

----------------------------------------------------------------------
Ran 42 tests in 0.332s

FAILED (failures=1)
ERROR: InvocationError for command /tmp/whisper/.tox/py39/bin/coverage run --branch '--include=whisper.py,test_whisper.py, bin/whisper-resize.py' test_whisper.py (exited with code 1)
_______________________________________________________________ summary _______________________________________________________________
ERROR:   py39: commands failed
$ . .tox/py39/bin/activate
(py39) $ coverage run --branch '--include=whisper.py,test_whisper.py, bin/whisper-resize.py' test_whisper.py
..........................................
----------------------------------------------------------------------
Ran 42 tests in 0.403s

OK
```

Does it pass for you when running via `tox`?

--- a/test_whisper.py
+++ b/test_whisper.py
@@ -819,7 +819,7 @@ class TestWhisper(WhisperTestBase):
         whisper.create(self.filename, retention)
 
         # insert data
-        now_timestamp = int((datetime.now() - datetime(1970, 1, 1)).total_seconds())
+        now_timestamp = int((datetime.utcnow() - datetime(1970, 1, 1)).total_seconds())
         now_timestamp -= now_timestamp % 60  # format timestamp
         points = [(now_timestamp - i * 60, i) for i in range(0, 60 * 24 * 2)]
         whisper.update_many(self.filename, points)

